.gitignore

```diff
@@ -17,3 +17,4 @@ ocean_lib.egg-info/
 .tox/
 consume-downloads/
 coverage.xml
+tests/resources/downloads/datafile.*
```

Comment:
Likely won't need this, see my comment below about pytest functionality for temporary directories.